src/mongo_types.ts

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable @typescript-eslint/ban-types */
+import type { Document, ObjectId } from './bson';
+
+/** @internal */
+export type TODO_NODE_2648 = any;
```

Comment:
When we get to revamping our operations layer, we can pipe through the generics, if this isn't our favorite way to document the TODO's we can refactor here.